README

```diff
@@ -1,18 +1,58 @@
+# perfgrind
+
 This is 'perfgrind', tools for collecting samples from Linux peformance events
 subsystem and converting profiling data to 'callgrind' format.
 
-Dependencies: elfutils (https://fedorahosted.org/elfutils/)
-
-Building:
-- create site.mak file and set FLAGS variable with paths to elfutils header and libraries
-  For example:
-
-FLAGS=-I/usr/local/elfutils/include -L/usr/local/elfutils/lib -O2 -march=native -Wl,-rpath /usr/local/elfutils/lib
-
-- make it
+Because of its own simplified format containing only the data nacessary for creating the callgrind profile the resulting fiel is commonly much smaller.
 
+# Usage
 
-Usage:
+## Overview
 - collect samples using 'pgcollect'
 - convert collected samples into 'callgrind' file using 'pgconvert'
 - open resulting 'callgrind' file in KCachegrind
+
+# `pgcollect` - collect samples
+Usage: `pgcollect outfile.pgdata [-z] [-F freq] [-s] {-p pid | [--] cmd}`
+
+Options to specify output:
+- `outfile.pgdata` name of output file
+- `-z` enable compression
```

Comment:
I'd prefer this to be added after implementation.

Reply:
Sure, that's one of the reasons for the draft.

When you like the general approach I'll finish the conversion to markdown and fix typos, then force-push without the -z.